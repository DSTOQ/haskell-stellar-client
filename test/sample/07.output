TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            GDKLIFHZGGSL24ZWIJHGQB46IL5DHLLI5Q7FV2AHIF42QWEGHOMQTEHL
        , _fee = FeeStroops { _feeStroops = 100 }
        , _seqNum = 46373869996474369
        , _timeBounds = Nothing
        , _memo = MemoNone
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    ChangeTrust
                      ChangeTrustOp
                        { _line =
                            AssetCreditAlphanum
                              HTKN GBV3M224ISSA7WBMISXGW3VV5TA4W6GNYBVGDNXYGOPMVO3RNVW5QBUF
                        , _limit = NonNegativeInt64 (Refined 9223372036854775807)
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint { _signatureHint = "863b9909" }
          , _signature =
              Signature
                { _signature =
                    "dba1af876fc7947dbd2ebc049828b33cc847438b2d1bfd39d0b207f780f3387b8b00bc539cab1ef22515b24afdd8bda9d92b5eeda427db78a37c6e23740b8d08"
                }
          }
      ]
  }