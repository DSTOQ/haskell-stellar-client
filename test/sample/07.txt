AAAAANS0FPkxpL1zNkJOaAeeQvozrWjsPlroB0F5qFiGO5kJAAAAZACkwMoAAAABAAAAAAAAAAAAAAABAAAAAAAAAAYAAAABSFRLTgAAAABrtmtcRKQP2CxErmtutezBy3jNwGphtvgznsq7cW1t2H//////////AAAAAAAAAAGGO5kJAAAAQNuhr4dvx5R9vS68BJgoszzIR0OLLRv9OdCyB/eA8zh7iwC8U5yrHvIlFbJK/di9qdkrXu2kJ9t4o3xuI3QLjQg=
