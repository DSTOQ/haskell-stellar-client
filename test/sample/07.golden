TransactionEnvelope {transaction = Transaction {sourceAccount = PublicKeyEd25519 {unPublicKeyEd25519 = 96208501124494311859351736218932660155904181775516288493633903600368474626313}, fee = Fee {unFee = 100}, seqNum = SequenceNumber {unSequenceNumber = 46373869996474369}, timeBounds = Nothing, memo = MemoNone, operations = VarLen {unVarLen = [Operation {sourceAccount = Nothing, body = ChangeTrust (ChangeTrustOp {line = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1213483854}) (PublicKeyEd25519 {unPublicKeyEd25519 = 48719781938602096453436114010251764830087310142640098156249894279754447416792}), limit = Just 9223372036854775807})}]}}, signatures = VarLen {unVarLen = [DecoratedSignature {hint = SignatureHint {unSignatureHint = 2252052745}, signature = Signature {unSignature = VarLen {unVarLen = "\219\161\175\135o\199\148}\189.\188\EOT\152(\179<\200GC\139-\ESC\253\&9\208\178\a\247\128\243\&8{\139\NUL\188S\156\171\RS\242%\NAK\178J\253\216\189\169\217+^\237\164'\219x\163|n#t\v\141\b"}}}]}}